### ISSUE TITLE
Historie & Wasserverbrauch "Widrige Umstände"

### ISSUE THREAD

**Issue by Ogrimgro** [Plushtoast/dsa5-foundryVTT#1523]
**Is your feature request related to a problem? Please describe.**
Die aktuelle Funktion bietet die Möglichkeit den Verbrauch von Nahrung zu simulieren und dadurch Hunger auszulösen. Eine analoge Funktion für Wasserverbrauch wäre wünschenswert.

**Describe the solution you'd like**
Ein zusätzliche Iten "Ration Wasser" und eine Verbrauchsfunktion analog zur Nahrung

**Describe alternatives you've considered**
Manuelles erstellen des Items und doppelte (1x Essen, 1x Trinken) Nutzung der vorhanden Funktion ist möglich, aber weniger schön weil nicht zwischen Nahrung und Wasser unterschieden wird. Dementsprechend kann theoretisch beides mit Nahrung "bezahlt" werden.

**Comment by Plushtoast** [comment]
Du kannst einfach selbst ausrüstung anlegen die wasser nennen und dann unter typ "Lebensmittel" einstellen.
Anschließend kannst du zb von deinen spielern jeweils eine wasser und normale ration verlangen oder 2x die ration auslösen; einmal für wasser und einmal für essen.
alles was in der kategorie lebensmittel zu finden ist, wird dort auswählbar angezeigt

**Comment by Ogrimgro** [comment]
Ja, steht ja oben schon in der Beschreibung und habe ich tatsächlich auch schon gemacht. Ist halt nicht ganz so schön weil man dann den gleichen Text hat und manuell darauf achten muss, dass das richtige "bezahlt" wird. Aber ich kann verstehen wenn man dafür den Aufwand nicht gehen will

**Comment by Plushtoast** [comment]
Das wäre zu wenig um eine sinnvolle Erweiterung zu sein. Wenn überhaupt müsste es dann noch speichern, wer wieviel rationen von was die letzten 10 tage beigetragen hat
